grunttasks/xss-mitigate.js

```diff
@@ -0,0 +1,78 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var xss = require('xss');
+
+module.exports = function (grunt) {
+  grunt.registerTask('xss-flag', 'Run the XSS parser on the generated l10n files', [
+    'l10n-json-to-html',
+    'xss-parse'
+    ]
+  );
+
+  grunt.registerMultiTask('xss-parse', 'Parse HTML files to check for XSS vectors', function () {
+    var templateRegex = /\%\(\w+\)s/;
```

Comment:
it looks for usage of templates like so:
%(sampleURI)
What do you think is a better name?